### DISCUSSION TITLE
GitHub Pages website is simply not updating

### DISCUSSION THREAD

**Discussion by SanchayanDutta** [community/community#23536] [Projects and Issues]
<p>Here’s the repository from which I’m generating my homepage:</p>
<p><a href="https://github.com/SanchayanDutta/sanchayandutta.github.io" class="inline-onebox" rel="noopener nofollow ugc">GitHub - SanchayanDutta/sanchayandutta.github.io: Github Pages template for academic personal websites, forked from mmistakes/minimal-mistakes</a>.</p>
<p>Since yesterday, I’ve been trying to update parts of my site. However, no matter how many times I commit the changes are simply not being reflected on the website.</p>
<p>For instance, I’ve tried adding two “Test.” sentences to the about.md file (/blob/master/_pages/about.md), but it doesn’t get reflected on the website (<a href="https://sanchayandutta.cs.ucdavis.edu" rel="noopener nofollow ugc">https://sanchayandutta.cs.ucdavis.edu</a>) at all even though it’s been several hours.</p>
<p>Can someone help me here?</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>I’m not yet checking the details of it, but you have build errors, with that, your site will not update,</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/c/a/ca027ce1285676123cde466c7a8cc4486e106574.png" data-download-href="/uploads/short-url/sP3Mj0YpXS8b2sMrSj55kElEo28.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181077780-a00387d1-30d9-4ebb-9d25-dda6f48b9642.png" alt="image" data-base62-sha1="sP3Mj0YpXS8b2sMrSj55kElEo28" width="564" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/c/a/ca027ce1285676123cde466c7a8cc4486e106574_2_564x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/c/a/ca027ce1285676123cde466c7a8cc4486e106574_2_846x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/c/a/ca027ce1285676123cde466c7a8cc4486e106574_2_1128x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/c/a/ca027ce1285676123cde466c7a8cc4486e106574_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1175×1040 62.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>the last time it was working was 8 hours ago, remember all the updates after that time so you can trace the problem/s</p>
<p><img src="https://user-images.githubusercontent.com/47092464/181077781-f7f98591-d472-4aea-9fc2-1aae26e3a4cb.png" alt="image" data-base62-sha1="o8xTOUlqAYWDkTFyDrPzt87yXQc" width="395" height="119"></p>

**Comment by SanchayanDutta** [comment]
<p>Thank you so much. I’m trying to trace the error. Is there any way to automatically restore to a previous version of the repository though?</p>

**Comment by jfullstackdev** [comment]
<p>this might be the one you’re looking for,</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://www.git-tower.com/learn/git/faq/restore-repo-to-previous-revision">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/47092464/181077786-08f8727c-ab36-4d74-96c3-57f31ea55e16.png" class="site-icon" width="16" height="16">

      <a href="https://www.git-tower.com/learn/git/faq/restore-repo-to-previous-revision" target="_blank" rel="noopener">git-tower.com</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/361;"><img src="https://user-images.githubusercontent.com/47092464/181077787-71a59d14-31ed-482f-8698-4d2e268ac9e4.png" class="thumbnail" width="690" height="361" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/1/815097487a63ff184fc8961acae2340b4098c741_2_690x361.png, https://aws1.discourse-cdn.com/github/optimized/3X/8/1/815097487a63ff184fc8961acae2340b4098c741_2_1035x541.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/8/1/815097487a63ff184fc8961acae2340b4098c741.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/1/815097487a63ff184fc8961acae2340b4098c741_2_10x10.png"></div>

<h3><a href="https://www.git-tower.com/learn/git/faq/restore-repo-to-previous-revision" target="_blank" rel="noopener">How can I restore a previous version of my project?</a></h3>

  <p>Git offers different ways to restore a previous state of your project files. Here's how you can return to an old revision.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

**Comment by SanchayanDutta** [comment]
<p>Awesome! Thank you so much for your help! I managed to locate the error in the build after all.</p>
<p>Wishing you a great day ahead. :slight_smile:</p>
